### PR TITLE
Adding documentation for actions for switchbot_cloud

### PR DESCRIPTION
**Pull request by SeraphicRav**
## Proposed change

The adds the documentations for a service to list devices and another one to directly send commands to them.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/121919
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
  - Added comprehensive documentation for the SwitchBot Cloud integration in Home Assistant. This includes setup instructions, prerequisites, supported devices, available services, and important API considerations.

### CODE REVIEW

File: source/_integrations/switchbot_cloud.markdown [hunk 1]
```diff
@@ -42,6 +42,29 @@ Please note, device names configured in the SwitchBot app are transferred into H
 - MeterPlus
 - Outdoor Meter
 
+## Services
```
Comment (c0ffeeca7) [inline source/_integrations/switchbot_cloud.markdown]:
```suggestion
## Actions
```
In this release, we are renaming services to actions

File: source/_integrations/switchbot_cloud.markdown [hunk 2]
```diff
@@ -42,6 +42,29 @@ Please note, device names configured in the SwitchBot app are transferred into H
...
+
+### switchbot_cloud.list_devices
+
+This service lists all the available devices for the SwitchBot API.
```
Comment (c0ffeeca7) [inline source/_integrations/switchbot_cloud.markdown]:
```suggestion
This action lists all the available devices for the SwitchBot API.
```

File: source/_integrations/switchbot_cloud.markdown [hunk 3]
```diff
@@ -42,6 +42,29 @@ Please note, device names configured in the SwitchBot app are transferred into H
...
+
+| Field name  | Description                                |
+| ----------- | ------------------------------------------ |
+| unique_id   | Id of the device used by SwitchBot.        |
```
Comment (c0ffeeca7) [inline source/_integrations/switchbot_cloud.markdown]:
```suggestion
| unique_id   | ID of the device used by SwitchBot.        |
```

File: source/_integrations/switchbot_cloud.markdown [hunk 4]
```diff
@@ -42,6 +42,29 @@ Please note, device names configured in the SwitchBot app are transferred into H
...
+| device_name | Name set in the SwitchBot application.     |
+| device_type | Type of the device according to SwitchBot. |
+
+### switchbot_cloud.send_command
+
+This service allows sending control command to devices as specified in [SwitchBot API documentation](https://github.com/OpenWonderLabs/SwitchBotAPI?tab=readme-ov-file#send-device-control-commands)
```
Comment (c0ffeeca7) [inline source/_integrations/switchbot_cloud.markdown]:
```suggestion
This action allows sending control commands to devices as specified in [SwitchBot API documentation](https://github.com/OpenWonderLabs/SwitchBotAPI?tab=readme-ov-file#send-device-control-commands)
```

File: source/_integrations/switchbot_cloud.markdown [hunk 5]
```diff
@@ -42,6 +42,29 @@ Please note, device names configured in the SwitchBot app are transferred into H
...
+
+| Service Data Attribute | Required | Description                                   |
+| ---------------------- | -------- | --------------------------------------------- |
+| unique_id              | yes      | SwitchBot unique id.                          |
```
Comment (c0ffeeca7) [inline source/_integrations/switchbot_cloud.markdown]:
```suggestion
| unique_id              | yes      | SwitchBot unique ID.                          |
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Tiny tweaks

File: source/_integrations/switchbot_cloud.markdown [hunk 6]
```diff
@@ -42,6 +42,29 @@ Please note, device names configured in the SwitchBot app are transferred into H
...
+| ----------- | ------------------------------------------ |
+| unique_id   | ID of the device used by SwitchBot.        |
+| device_name | Name set in the SwitchBot application.     |
+| device_type | Type of the device according to SwitchBot. |
+
+### switchbot_cloud.send_command
+
+This action allows sending control commands to devices as specified in [SwitchBot API documentation](https://github.com/OpenWonderLabs/SwitchBotAPI?tab=readme-ov-file#send-device-control-commands)
+
+| Service Data Attribute | Required | Description                                   |
```
Comment (c0ffeeca7) [inline source/_integrations/switchbot_cloud.markdown]:
```suggestion
| Data attribute | Required | Description                                   |
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
forgot one

**Comment by SeraphicRav** [comment]
@c0ffeeca7 Thanks for the review!

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @SeraphicRav :+1:

**Comment by SeraphicRav** [comment]
After a discussion with Joostlek on Discord, we decided that for UX reasons, we should implement actions in entities, I will close this PR.

**Pull request closed without merging**